docs/index_resources.md

```diff
@@ -0,0 +1,138 @@
+# Getting a Collection of Resources
+
+To get a paginated list of Books, we will use **indexAction** of our controller.
+In the default scenario, it will return an instance of paginator, with a list of Books.
+
+```yaml
+    # config/routes.yaml
+
+    app_book_index:
+        path: /books
+        methods: [GET]
+        defaults:
+            _controller: app.controller.book:indexAction
+```
+When you go to ``/books``, the ResourceController will use the repository (``app.repository.book``) to create a paginator.
+The default template will be rendered - ``App:Book:index.html.twig`` with the paginator as the ``books`` variable.
+
+A paginator can be a simple array, if you disable the pagination, otherwise it is an instance of ``Pagerfanta\Pagerfanta``
+which is a [Library](https://github.com/whiteoctober/Pagerfanta) used to manage the pagination.
+
+## Overriding the Template and Criteria
+
+Just like for the **showAction**, you can override the default template and criteria.
+
+```yaml
+    # config/routes.yaml
+
+    app_book_index_inactive:
+        path: /books/disabled
+        methods: [GET]
+        defaults:
+            _controller: app.controller.book:indexAction
+            _sylius:
+                filterable: true
+                criteria:
+                    enabled: false
+                template: Book/disabled.html.twig
+```
+This action will render a custom template with a paginator only for disabled Books.
+
+## Sorting
+
+Except filtering, you can also sort Books.
+
+```yaml
+
+    # config/routes.yaml
+
+    app_book_index_top:
+        path: /books/top
+        methods: [GET]
+        defaults:
+            _controller: app.controller.book:indexAction
+            _sylius:
+                sortable: true
+                sorting:
+                    score: desc
+                template: Book/top.html.twig
+```
+Under that route, you can paginate over the Books by their score.
+
+## Using a Custom Repository Method
+
+You can define your own repository method too, you can use the same way explained
+in [show_resource](http://docs.sylius.com/en/latest/components_and_bundles/bundles/SyliusResourceBundle/show_resource.html#using-custom-repository-methods).
+
+### **Note**
+If you want to paginate your resources you need to use ``EntityRepository::getPaginator($queryBuilder)``.
+It will transform your doctrine query builder into ``Pagerfanta\Pagerfanta`` object.
+
+## Changing the "Max Per Page" Option of Paginator
+
+You can also control the "max per page" for paginator, using ``paginate`` parameter.
+
+```yaml
+    # config/routes.yaml
+
+    app_book_index_top:
```

Comment:
I would say, that YAML files shouldn't be indented

Reply:
That's true for all blocks (bash, yaml, PHP)

Reply:
Agreed. It makes pasting the code a little harder.